### DISCUSSION TITLE
how to use useVuePdfEmbed ?

### DISCUSSION THREAD

**Discussion by thethreedays** [hrynko/vue-pdf-embed#230] [Q&A] [answered]
How to use this method and why I can't get the total number of pages
![image](https://github.com/user-attachments/assets/2eb5af7b-2087-4bc9-b87f-06d28eea4253)

**Comment by hrynko** [comment, marked as answer]
Hi @thethreedays,

The following is the minimal example of using the `useVuePdfEmbed` composable and displaying the number of pages:

```vue
<script setup>
import VuePdfEmbed, { useVuePdfEmbed } from 'vue-pdf-embed'

const { doc } = useVuePdfEmbed({ source: '<PDF_SOURCE>' })
</script>

<template>
  <p v-if="doc">Pages: {{ doc.numPages }}</p>
  <VuePdfEmbed :source="doc" />
</template>
```
